sandbox.html

```diff
@@ -244,7 +251,8 @@
         layers: layers ? JSON.parse(layers) : undefined,
         useMapSrs: useMapSrs === 'on',
         fullGeometries: fullGeometries === 'on',
-        zoomTarget: zoomToGeometry === 'on' ? { type: 'geometry' } : undefined
+        zoomTarget: zoomToGeometry === 'on' ? { type: 'geometry' } : undefined,
+        buffer: distance ? { value: Number(distance), lengthUnit: lengthUnit ? lengthUnit : 'm' } : undefined
```

Comment:
I meant that
```javascript
lengthUnit: lengthUnit ? lengthUnit : 'm'
```
can be simplified to
```javascript
lengthUnit
```
because the `lengthUnit` cannot be undefined.